java/src/IceGridGUI/src/main/java/com/zeroc/IceGridGUI/LiveDeployment/MetricsViewEditor.java

```diff
@@ -184,7 +184,7 @@ public void valueChanged(TreeSelectionEvent e) {
             _properties.load("metrics.cfg");
             sectionSort.addAll(
                 Arrays.asList(
-                    _properties.getIcePropertyAsList("IceGridGUI.Metrics")));
+                    _properties.getPropertyAsList("IceGridGUI.Metrics")));
 
             String metricsDefs =
                 coord.getProperties().getIceProperty("IceGridAdmin.MetricsConfigs");
```

Comment:
I'm not sure we allow setting IceGridAdmin props right now in the GUI (we should double check). Additionally, I can't find any documentation for this property. 

[MetricsConfig](https://doc.zeroc.com/pages/viewpage.action?pageId=18256255)